src/main/java/com/blackhearth/blockchain/params_reader/Pair.java

```diff
@@ -0,0 +1,11 @@
+package com.blackhearth.blockchain.params_reader;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@AllArgsConstructor
+@Data
+public class Pair<K, V> {
```

Comment:
Chyba istnieje coś takiego w pakiecie spring.cloud, można pomyśleć o imporcie. (Ale jeśli używamy tylko Pair i nic poza tym to nie ma sensu).